### PR TITLE
Add GraphQL plugin

### PR DESCRIPTION
**Pull request by Kocal**
## Rollup Plugin Name: `graphql`

This PR contains:

- [ ] bugfix
- [x] feature
- [ ] refactor
- [ ] documentation
- [ ] other

Are tests included?

- [x] yes (_bugfixes and features will not be merged without tests_)
- [ ] no

Breaking Changes?

- [ ] yes (_breaking changes will not be merged unless absolutely necessary_)
- [x] no

If yes, then include "BREAKING CHANGES:" in the first commit message body, followed by a description of what is breaking. 

List any relevant issue numbers:

### Description

Following https://github.com/rollup/plugins/issues/2#issuecomment-557902616 and https://github.com/rollup/rollup/issues/3793#issuecomment-697290047, this PR import and migrate [`@kocal/rollup-plugin-graphql`](https://github.com/Kocal/rollup-plugin-graphql) under `@rollup/plugin-graphql` package.

I will deprecate my fork once this PR will be merged and released :+1:

### CODE REVIEW

**Comment by shellscape** [comment]
thanks!

I'll make time for this and the ESLint plugin addition this week, I'm determined.

**Comment by Kocal** [comment]
Great! :D

File: packages/graphql/src/toESModules.js [hunk 1]
```diff
@@ -0,0 +1,42 @@
+import { EOL } from 'os';
+
+export default function(source) {
```
Comment (NotWoods) [inline packages/graphql/src/toESModules.js:3]: Could you add the function name here so its a bit easier to read on its own? JSDoc would be nice to have.
```suggestion
export default function toESModules(source) {
```

File: packages/graphql/src/toESModules.js [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+  }
+
+  return replaceRequires(replaceModuleExports(source));
+}
+
+function replaceModuleExports(source) {
+  return source
+    .replace('module.exports = doc', 'export default doc')
+    .replace(
+      /module\.exports\["(.*)"] = oneQuery\(doc, "(.*)"\)/g,
```
Comment (NotWoods) [inline packages/graphql/src/toESModules.js:15]: We should have some comment explaining what the regex is doing.

Comment (Kocal) [reply packages/graphql/src/toESModules.js:15]: I think the name of the function `replaceModuleExports` already does the job. What do you think?

Comment (NotWoods) [reply packages/graphql/src/toESModules.js:22]: Fair enough :+1:

**Comment by shellscape** [comment]
<< narrator: he failed to make time >>

My apologies, folks. I have not forgotten about this one and the ESLint plugin. It will be done, I promise.

**Comment by Kocal** [comment]
Don't worry :)

**Comment by Kocal** [comment]
:eyes:

**Pull request merged**